packages/cli/src/oclif/commands/promote.js

```diff
@@ -117,7 +123,14 @@ class PromoteCommand extends BaseCommand {
   }
 }
 
-PromoteCommand.flags = buildFlags();
+PromoteCommand.flags = buildFlags({
+  commandFlags: {
+    'skip-changelog': flags.boolean({
+      description:
+        'Suppress prompts asking to confirm a corresponding changelog entry for the version you are attempting to promote. ',
```

Comment:
`skip-changelog` sounds like we're skipping changelog, even if there's one, while the actual behavior is to automatically answer "yes" to any prompts.jj

I suggest we name this flag `--yes` and `-y`, like how [`apt-get`](https://linux.die.net/man/8/apt-get) does it:

```suggestion
    yes: flags.boolean({
      char: 'y',
      description:
        'Automatically answer "yes" to any prompts. Useful if you want to avoid interactive prompts to run this command in CI.',
    }),
```

Reply:
@eliangcs I'm cool with approach 👍